README.md

```diff
@@ -43,6 +43,41 @@ Running Edge in Docker with our recommended setup:
 docker run -it -p 3063:3063 -e STRICT=true -e UPSTREAM_URL=<yourunleashinstance> unleashorg/unleash-edge:<mostrecentversion> edge
 ```
 
+## Strict behavior (Since 19.2.0)
+
+If started with the `--strict` flag or the `STRICT` environment variable, Edge now starts with strict behavior and must
+be
+given
+tokens at startup.
+Edge will lock down access for new tokens that have a wider or different access scope than the tokens it was started
+with.
+
+E.g. if you start with one wildcard token with access to development `*:development.<somelongrandomstring` and later a
+client comes with a
+token accessing a different environment, say production `*:production.<somedifferentrandomstring>` Edge will return 403,
+because that would change the scope of access.
+
+Strict behavior still allows using tokens (both frontend and client) with narrower access than what the tokens it was
+started with has.
+
+E.g. If you start with one wildcard token with access to the development
+environment `*:development.<somelongrandomstring>` and your clients use various tokens with access to specific projects
+in the development environment, Edge will filter features to only grant access to the narrower scope.
+
+## Dynamic behavior
+
+With dynamic behavior, Edge behaves as it has since v1.0.0, any new client tokens are validated against upstream and if
+found
+to be valid, a refresh job will be configured with the minimum set of tokens that are able to fetch all projects and
+environments Edge has seen. Set with `--dynamic` or the `DYNAMIC` environment variable. (19.2.0 / July 4th 2024): We're
+looking to deprecate this
+behavior. If your company needs this behavior, reach out to us on Slack, the final decision has not been made yet. In
+19.2.0
+this behavior is still the default, but Edge will log that you should make a choice between dynamic or strict behavior.
+If
+you
+explicitly choose dynamic behavior, Edge will warn about the planned deprecation.
```

Comment:
Even if you don't explicitly choose it and we default to it, we still show the warning. It behaves as shown in the PR description of https://github.com/Unleash/unleash-edge/pull/476

Reply:
That's true.

Reply:
Agreed. Changed text to `When dynamic behavior is selected (by default or by choice), Edge will print a warning about the planned deprecation`